src/example-button.module/meta.json

```diff
@@ -0,0 +1,7 @@
+{
+  "label": "Example button",
+  "host_template_types": ["PAGE", "BLOG_LISTING", "BLOG_POST"],
+  "icon": "",
```

Comment:
We do require icons, should there be one here?

Reply:
Yes! I was just waiting on feedback from the review team about how providers typically include them for standalone modules. It sounds like it is normally absolute URLs so going to add one in now.